laterpay-migrator/asset_sources/scss/objects/_lists.scss

```diff
@@ -0,0 +1,13 @@
+.lp_list,
```

Comment:
Selector `lp_list` should be written in lowercase with hyphens<br>Selector `lp_list__item` should be written in lowercase with hyphens